internal/server/utils/redisCmdAdapter.go

```diff
@@ -252,3 +254,58 @@ func isBase64Encoded(s string) bool {
 	}
 	return false
 }
+
+func unmarshalRequestBody(data []byte, v *map[string]interface{}) error {
+	var rawMap map[string]interface{}
+
+	if err := json.Unmarshal(data, &rawMap); err != nil {
+		return err
+	}
+
+	for key, val := range rawMap {
+		switch val := val.(type) {
+		case float64:
+			// force check whether the float64 value is a big integer
+			// if it is, typecast it and set the value for the key in the rawMap
+			if val == float64(int64(val)) {
+				rawMap[key] = json.Number(strconv.FormatInt(int64(val), 10))
+			}
+		case map[string]interface{}:
+			jsonValue, err := json.Marshal(val)
+			if err != nil {
+				return err
+			}
+			// recursively unmarshal nested JSON body
+			var nestedMap map[string]interface{}
+			if err := unmarshalRequestBody(jsonValue, &nestedMap); err != nil {
+				return err
+			}
+			rawMap[key] = nestedMap
+		case []interface{}:
+			for i, item := range val {
+				if nestedMap, ok := item.(map[string]interface{}); ok {
+					jsonValue, err := json.Marshal(nestedMap)
+					if err != nil {
+						return err
+					}
+					// recursively unmarshal nested JSON body
+					var nestedNestedMap map[string]interface{}
+					if err := unmarshalRequestBody(jsonValue, &nestedNestedMap); err != nil {
+						return err
+					}
+					val[i] = nestedNestedMap
+				}
+			}
+		}
+	}
+
+	bytes, err := json.Marshal(rawMap)
```

Comment:
```suggestion
	bt, err := json.Marshal(rawMap)
```

Reply:
Consider naming it something else instead of bytes so that we don't need to create alias for the bytes package.

Reply:
Makes sense, fixed this.